Makefile

```diff
@@ -77,6 +87,11 @@ clean:
 	@$(RM) $(SRCS_TSLINT_RULES_JS)
 
 
+$(SHELLCHECK_TARGET): $(SRCS_SHELL)
+	@$(RM) $(SHELLCHECK_TARGET)
+	@$(SHELLCHECK) $(SRCS_SHELL)
+	@$(TOUCH) $(SHELLCHECK_TARGET)
```

Comment:
Since `shellcheck` doesn't produce any output, this is the target that gets generated on success.  This way, we only re-run the target if the shell files are newer than the dummy target.